server/functions/basebuilding/fn_building_create.sqf

```diff
@@ -89,6 +89,13 @@ _building setVariable ["para_g_objects", _objects, true];
 //Do NOT do an object update, as we've instantiated everything correctly above.
 [_building, _buildProgress, false] call para_s_fnc_building_add_build_progress;
 
+//Block master arm from using buildings as a supply point.
+{
+	_x setVariable ["vn_master_arm_supplyAmmo", false, true];
+	_x setVariable ["vn_master_arm_supplyFuel", false, true];
+	_x setVariable ["vn_master_arm_supplyRepair", false, true];
+}forEach _objects;
```

Comment:
this might fail to broadcast... I'd recommend using publicVariable instead of relying on setVariable to broadcast it alone.

Reply:
The variable is not in the missionNamespace, so my understanding is that publicVariable is not usable here

Reply:
Ah you are correct... I was under the impression publicVariable would work for any variable due to this working
`RandomValue = 1
publicVariable "RandomValue"`